apps/dashboard/src/app/dashboard/components/schools/schools.component.ts

```diff
@@ -28,6 +30,7 @@ export class SchoolsComponent implements OnInit, OnDestroy {
   opened = false;
   updating = false;
   public school: School;
+  public users: User [] = []
```

Comment:
Do you need this public var? 
```suggestion
  users: User [] = []
```